src/obfl-specification.html

```diff
@@ -2618,6 +2618,20 @@ <h3 id="block-atts">Block</h3>
     <dd>Indent of every line in the block except the first one (in
     characters)</dd>
     <dd>A number.</dd>
+    <dd>This attribute is used to create hanging indentation.</dd>
+  <dt>right-last-line-indent [optional]</dt>
+    <dd>Right indent of the last line of a block (in characters)</dd>
+    <dd>A number.</dd>
+  <dt>right-text-indent [optional]</dt>
+    <dd>Right indent of every line in the block except the last one
+    (in characters)</dd>
```

Comment:
Why "except the last one"? That seems unnatural to me.
I would go for "including the last one" and with `right-last-line-indent` you can override this value if you like.

Reply:
Symmetry with `text-indent`.